ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/sno/lvms/cucushift-installer-rehearse-aws-ipi-sno-lvms-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-sno-lvms
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-sno-provision
```

Comment:
General question, for LVMS on SNO we need  vm_instance with 8 vCpu, I usually overwrite vm_type: 'm6i.2xlarge' to setup OCP AWS flexy cluster, do we have this vm_type setup in this chain?

Reply:
Good point, currently in this chain we don't overwrite vm_type, I'll overwrite it in our workflow.